src/translations/screens/subscreens/TravelToken.ts

```diff
@@ -4,84 +4,100 @@ const SelectTravelTokenTexts = {
+      'Switch between phone / t:card',
     ),
     faq: {
       title: _('Ofte stilte spørsmål', 'Frequently asked questions'),
     },
     faqs: [
       {
-        question: _('Hva om jeg mister mobilen eller t:kortet?', ''), // TODO
+        question: _(
+          'Hva skjer om jeg mister mobilen eller t:kortet?',
+          'What happens if I lose my phone or t:card?',
+        ),
         answer: _(
-          'Lorem ipsum dolor sit amet, consectetur adipiscing elit.', // TODO
-          'Lorem ipsum dolor sit amet, consectetur adipiscing elit.', // TODO
+          'Du kan enkelt velge å bruke billetten din på et annet t:kort eller mobil. Det gjør du på **Min profil**.',
+          'You can easily head over to **My profile** to switch and use your ticket on another t:card or phone.',
         ),
       },
       {
-        question: _('Kan jeg reise med t:kort og mobil samtidig?', ''), // TODO
+        question: _(
+          'Kan jeg bruke billetten på både t:kort og mobil samtidig?',
+          'Can I use the ticket on both t:card and phone at the same time?',
+        ),
         answer: _(
-          'Lorem ipsum dolor sit amet, consectetur adipiscing elit.', // TODO
-          'Lorem ipsum dolor sit amet, consectetur adipiscing elit.', // TODO
+          'Nei. Billetten kan bare brukes på en dings om gangen, og billetten kan ikke deles med andre.',
+          'No. You can only use the ticket on one gadget at a time and the ticket can not be shared with others.',
         ),
       },
       {
-        question: _('Kan jeg reise uten mobil eller t:kort?', ''), // TODO
+        question: _(
+          'Kan jeg reise uten t:kort eller mobil?',
+          'Can I travel without my t:card or phone?',
+        ),
         answer: _(
-          'Lorem ipsum dolor sit amet, consectetur adipiscing elit.', // TODO
-          'Lorem ipsum dolor sit amet, consectetur adipiscing elit.', // TODO
+          'Nei. Du mpå alltid ha med deg den dingsen du har valgt å bruke billetten på.',
+          'No. You must always bring the gadget you gave chosen to use your ticket on.',
+        ),
+      },
+      {
+        question: _(
+          `Har du flere spørsmål om billettkjøp?`,
+          `Do you have any other questions about ticket purchases?`,
+        ),
+        answer: _(
+          `Se flere måter å kontakte oss på atb.no/kontakt`,
+          `See how to contact us at atb.no/en/contact`,
         ),
       },
     ],
   },
   toggleToken: {
-    title: _('Bytt mellom mobil / t:kort', 'Switch between mobile / t:card'), // TODO
+    title: _('Bytt mellom mobil / t:kort', 'Switch between phone / t:card'),
     radioBox: {
       tCard: {
         title: _('T:kort', 'T:card'),
         description: _(
           'Les av t:kortet ditt på holdeplass eller ombord. Kortet leses av ved kontroll.',
-          'TODO', // TODO
+          'Scan your t:card at the stop place or on board. The card will be scanned for ticket inspection',
```

Comment:
Endre til: 
Validate your t:card at the bus stop or on board. The card will be scanned in a ticket inspection.

Reply:
Denne brukes lenger ned også, så endre der i tillegg.

Reply:
:+1: